packages/database/src/entities/Project.ts

```diff
@@ -23,6 +23,9 @@ export class Project extends Node<Project> {
   @Property({ columnType: 'text' })
   name: string;
 
+  @Property({ columnType: 'text', nullable: true })
```

Comment:
AC didn't say this was supposed to be optional 😬 Your schema also has it as required

```suggestion
  @Property({ columnType: 'text' })
```

Reply:
Do we not need to leave the option open for a project to not have a repoLink right off of the bat? This was my call

Reply:
Right now we have no `PUT` endpoint so the project creation needs to be final